lib/formatio.js

```diff
@@ -58,6 +58,10 @@ function ascii(f, object, processed, indent) {
         return processed || quote ? "\"" + object + "\"" : object;
     }
 
+    if (typeof object === "symbol") {
+        return object.toString();
```

Comment:
Can you add a separate test for this case please?

Reply:
Done!